configure.ac

```diff
@@ -1510,10 +1510,10 @@ AC_CONFIG_FILES([Makefile
                  tests/API/XCCDF/variable_instance/Makefile
 
                  tests/schemas/Makefile
-		tests/bz2/Makefile
-		tests/codestyle/Makefile
-		tests/oval_details/Makefile
-		tests/nist/Makefile
+                 tests/bz2/Makefile
+                 tests/codestyle/Makefile
+                 tests/oval_details/Makefile
+                 tests/nist/Makefile
 
                  src/SCE/Makefile
                  tests/sce/Makefile])
```

Comment:
No problem. I would do the same thing to make it look prettier but the constant merge conflicts just aren't worth it :-) As a general rule of thumb most people will change indents to the right indents if they are also changing that particular line.